test/remote/gateways/remote_decidir_test.rb

```diff
@@ -360,14 +360,4 @@ def test_transcript_scrubbing
     assert_scrubbed(@credit_card.verification_value, transcript)
     assert_scrubbed(@gateway_for_purchase.options[:api_key], transcript)
   end
-
-  def test_transcript_scrubbing_network_token
```

Comment:
Removed this since cryptogram in Sandbox doesn't return the '/' in front of cryptogram value